js/crm.backdrop.js

```diff
@@ -0,0 +1,14 @@
+// http://civicrm.org/licensing
+CRM.$(function($) {
+  $(document).on('crmLoad', function(e) {
+    $('#admin-bar').css('display', 'none');
+  });
+  $('.crm-hidemenu').click(function(e) {
+    $('#admin-bar').css('display', 'block');
+  });
+  $('#crm-notification-container').click(function(e) {
```

Comment:
Better to do `$('#crm-notification-container').on('click', '#crm-restore-menu', function() {`